### ISSUE TITLE
Sort and per page broken aria

### ISSUE THREAD

**Issue by sensei100** [projectblacklight/blacklight#2220]
In conducting an accessibility review of our site recently, I noticed that with the Wave tool I am getting errors for both the sort by relevance and per page dropdown menus.  The error says: 
Broken ARIA menu - An ARIA menu does not contain required menu items. I see that the same errors are present on the demo Blacklight site as well.

**Comment by camillevilla** [comment]
hi @sensei100 and @dkinzer 👋. Thanks for filing! Just want to confirm: What version of Blacklight is your site using? I want to make sure I understand whether this needs to get backported to 6 or is something that needs resolution in 7. 

It looks like a fix for the sort menu was backported to Blacklight 6.x in #2219, but this still needs to be fixed in the [per page widget](https://github.com/projectblacklight/blacklight/blob/release-6.x/app/views/catalog/_per_page_widget.html.erb).

**Comment by sensei100** [comment]
@camillevilla We are using Blacklight 7.

**Comment by jkeck** [comment]
I notice that while that was "backported", this fix was never applied to master.

I believe we would need to add `role="menuitem"` to the links in master https://github.com/projectblacklight/blacklight/blob/8f3f2ea42350a1931d43dde0028a02c3c29f23b8/app/views/catalog/_sort_widget.html.erb#L9 and https://github.com/projectblacklight/blacklight/blob/8f3f2ea42350a1931d43dde0028a02c3c29f23b8/app/views/catalog/_per_page_widget.html.erb#L9

**Comment by dkinzer** [comment]
@camillevilla , actually this is all @sensei100 ... I had nothing to do with it, so all thanks go to her :D
